index.html

```diff
@@ -2175,6 +2232,116 @@ <h3>Verify Proof Sets and Chains</h3>
         </ol>
       </section>
 
+      <section class="normative">
+        <h3>Context Validation</h3>
+
+        <p>
+This section contains an algorithm that applications MUST run after running the
+algorithm in Section [[[#verify-proof]]] or Section
+[[[#verify-proof-sets-and-chains]]], when validating a [=conforming secured
+document=]. This algorithm takes a document ([=map=] |inputDocument|) and a set
```

Comment:
What is the reason for this order? Instinctively, my choice would be to first check whether the document is all right in terms of JSON-LD, ie, run the context validation, _before_ running it through the validating steps (that may also involve running RDFC on the RDF graphs generated by the JSON-LD processing).

(Note if the order is reversed, it also has to be updated in the new processing model section.)

Reply:
Fixed in 3107ccb1a830ae90efa417df7ccf4d735021bcb6.